### PR TITLE
SOC-2730 | FE | Adjustments for automation tests

### PR DESCRIPTION
**Pull request by rafalkalinski**
## Links

https://wikia-inc.atlassian.net/browse/SOC-2730

## Description

Changes in the components markup (add clear classes) for automation tests

## Reviewers

@bkoval @mleczey

### CODE REVIEW

File: front/main/app/components/modal-dialog.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-	classNameBindings: ['type', 'closeOnOverlayClick::layover-cursor-auto'],
+	classNameBindings: ['type', 'closeOnOverlayClick::layover-cursor-auto', 'isVisible', 'additionalClass'],
```
Comment (bkoval) [inline front/main/app/components/modal-dialog.js]: Why do we need to bind "isVisible'?

File: front/main/app/services/modal-dialog.js [hunk 2]
```diff
@@ -1,40 +1,55 @@
...
-				confirmButtonText,
-				header,
-				isDisplayed: true,
-				message,
-			});
+	display(displayParameters) {
+		if (!this.get('isDisplayed') && displayParameters.message !== this.get('message')) {
+			const params = {};
+
+			Object.assign(
```
Comment (bkoval) [inline front/main/app/services/modal-dialog.js]: What is it? ;)

Comment (rafalkalinski) [inline front/main/app/services/modal-dialog.js]: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

**Comment by bkoval** [comment]
LGTM!

**Pull request merged**